webapp/sources/rudder/rudder-rest/src/main/scala/com/normation/rudder/rest/RestDataSerializer.scala

```diff
@@ -549,6 +555,16 @@ final case class RestDataSerializerImpl (
     )
   }
 
+  def serializeHealthcheckResult(check: HealthcheckResult): JValue = {
+    val status = check match {
+      case c:Critical => "Critical"
+      case o:Ok => "Ok"
```

Comment:
if you don't use a variable, don't bind it to a name (so here: `case _:OK =>` ...)